chunky3d/chunky.py

```diff
@@ -614,7 +618,7 @@ def set_chunk(self, idx, val):
                 self._grid[idx] = d
 
     def run_multivariate(self, func, sparse_list, *args):
-        keys = set()
```

Comment:
this was needed for .mul() but I think it fixes some other operators too. 
Before that - the set of iterated keys was just taken from the 'other' sparses. and hence 'active' chunks from current 'Sparse' weren't taken into account at all